src/bdp.cpp

```diff
@@ -44,6 +44,13 @@ using BloombergLP::blpapi::Message;
 using BloombergLP::blpapi::MessageIterator;
 using BloombergLP::blpapi::Name;
 
+namespace bbg = BloombergLP::blpapi;
+
+namespace {
+    const bbg::Name RESPONSE_ERROR("responseError");
+}
```

Comment:
This looks a little verbose and out of sync with the rest,   Could this not have been done via the existing `using BloombergLP::blpapi::Name;` from line 45 along with

```c++
const Name Response_Error("responseError");
```

inside the function?

Reply:
@johnlaing Can you maybe take a look on your end too?

Reply:
I looked at what was done in getTicks/getBars but yes it is not exactly the same thing in this part of code.

Reply:
That looks better.  

@johnlaing Would be great if you could give the pr a spin as well.